setup.cfg

```diff
@@ -3,5 +3,5 @@ universal = 1
 
 [flake8]
 ignore =
-max-line-length = 79
+max-line-length = 88
```

Comment:
nice .)

Reply:
This is a pre-cursor to just turning on black ;)